larana/OpticalDetector/SimPhotonCounter_module.cc

```diff
@@ -594,105 +594,141 @@ namespace opdet {
       //art::Handle< std::vector<sim::SimPhotonsLite> > photonHandle;
       //evt.getByLabel(mod, photonHandle);
 
-	// Loop over direct/reflected photons
-	for (auto const& ph_handle: photon_handles) {
+        // Loop over direct/reflected photons
+        for (auto const& ph_handle: photon_handles) {
           // Do some checking before we proceed
           if (!ph_handle.isValid()) continue;
           if (ph_handle.provenance()->moduleLabel() != mod) continue;   //not the most efficient way of doing this, but preserves the logic of the module. Andrzej
 
           bool Reflected = (ph_handle.provenance()->productInstanceName() == "Reflected");
 
+          //Reset counters
+          fCountEventAll=0;
+          fCountEventDetected=0;
 
-
-      //Reset counters
-      fCountEventAll=0;
-      fCountEventDetected=0;
-
-      if(fVerbosity > 0) std::cout<<"Found OpDet hit collection of size "<< (*ph_handle).size()<<std::endl;
+          if(fVerbosity > 0) std::cout<<"Found OpDet hit collection of size "<< (*ph_handle).size()<<std::endl;
 
 
-      if((*ph_handle).size()>0)
-      {
-
-        for ( auto const& photon : (*ph_handle) )
-        {
-          //Get data from HitCollection entry
-          fOpChannel=photon.OpChannel;
-          std::map<int, int> PhotonsMap = photon.DetectedPhotons;
+          if((*ph_handle).size()>0)
+          {
 
-          //Reset Counters
-          fCountOpDetAll=0;
-          fCountOpDetDetected=0;
-	  fCountOpDetReflDetected=0;
+            for ( auto const& photon : (*ph_handle) )
+            {
+              //Get data from HitCollection entry
+              fOpChannel=photon.OpChannel;
+              std::map<int, int> PhotonsMap = photon.DetectedPhotons;
 
-         for(auto it = PhotonsMap.begin(); it!= PhotonsMap.end(); it++)
-           {
-             // Calculate wavelength in nm
-	     if (Reflected) {
-		fWavelength = 450;
-	     }
-	     else {
-	     fWavelength= 128;   // original
-	     }
+              //Reset Counters
+              fCountOpDetAll=0;
+              fCountOpDetDetected=0;
+              fCountOpDetReflDetected=0;
 
-             //Get arrival time from phot
-             fTime= it->first;
-             //std::cout<<"Arrival time: " << fTime<<std::endl;
+              for(auto it = PhotonsMap.begin(); it!= PhotonsMap.end(); it++)
+              {
+                // Calculate wavelength in nm
+                if (Reflected) {
+                  fWavelength = 450;
+                }
+                else {
+                  fWavelength= 128;   // original
```

Comment:
Really minor point but could these hardcoded numbers be set as const data members at the start of the file as it would make maintenance easier?
e.g.
```
private:
   const float kReflectedWavelength = 450;
...
if (Reflected) fWavelength = kReflectedWavelength;
```

Reply:
Can do.

Commit 27c8c1e5.